jdaviz/core/helpers.py

```diff
@@ -35,3 +35,41 @@ def __init__(self, app=None):
 
     def load_data(self, data, parser_reference=None, **kwargs):
         self.app.load_data(data, parser_reference=parser_reference, **kwargs)
+
+    @property
+    def fitted_models(self):
+        """
+        Returns the fitted model parameters.
+
+        Returns
+        -------
+        parameters : dict
+            dict of Quantity arrays, or None.
+        """
+        MAX_DIMENSIONS = 5
+        all_models = {}
+
+        for dim in range(1, MAX_DIMENSIONS):
+            attrname = f'_fitted_{dim}d_models'
+            model = getattr(self.app, attrname, None)
+            if model:
+                all_models = {**all_models, **model}
```

Comment:
I am not still very sure what the intention is here. Do you want to return on first match, get only the last match, or collate all matches?

Reply:
Collate all matches.

Reply:
Maybe this functionality does not need to exist. What do you think @eteq ?

Reply:
I just don't see how it is collating with `=` assignment at every iteration. Do you mean to update the dictionary instead?

Reply:
Oh wait... you are passing in itself on the right... Nvm... I think maybe using `.update()` method is cleaner, not sure.

Reply:
You're right, much easier to understand.